tests/storage/snapshot.go

```diff
@@ -1293,49 +1293,59 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 
 				Expect(snapshot.Status.CreationTime).To(BeNil())
 			})
+		})
 
-			It("vmsnapshot should update error if vmsnapshotcontent is unready to use and error", func() {
-				vm = tests.RunVMAndExpectLaunchWithRunStrategy(virtClient, vm, v1.RunStrategyAlways)
```

Comment:
@Barakmor1 Don't you need to wait for the VM to be ready? like in `RunVMAndExpectLaunchWithRunStrategy`
```
Eventually(ThisVM(updatedVM)).WithTimeout(300 * time.Second).WithPolling(time.Second).Should(BeReady())
```

Reply:
Yes, Thanks for the note will update the commit